dojo/models.py

```diff
@@ -37,7 +37,7 @@
 from django.db.models import JSONField
 import hyperlink
 from cvss import CVSS3
-from dojo.settings.settings import SLA_BUSINESS_DAYS
+from dojo.settings.settings import SLA_BUSINESS_DAYS, ENABLE_AUDITLOG
```

Comment:
I know this was here before you added to it, but I must ask anyways
>It would be better to use the registered version in django.conf.settings rather than importing from the file directly

Reply:
Done